### ISSUE TITLE
[Feature Request] slimming pokestop icon

### ISSUE THREAD

**Issue by RezzZ** [PokeAlert/PokeAlert#433]
currently the pokestop icon is taking up a lot of screen real-estate when enabled. Any chance we can slim down the icon to a simple blue/purple dot (without the cube on top)? That will cleanup the screen nicely and still leave enough room for found pokemon.
Same can be said about the gym icon..

**Comment by PokeAlert** [comment]
If anyone can provide me with slider icon I'll put them instead. I'm really not good with assets things

**Comment by RezzZ** [comment]
what format would like like the icon in? png file? A simple blue/purple dot seems more then enough.

**Comment by arhi3a** [comment]
It doesn't have to be something fancy, it can be simpliest red/blue/yellow circle with number (gym level). It will be still easy to distinguish it from pokemon icons.

**Comment by arhi3a** [comment]
![gym red](https://cloud.githubusercontent.com/assets/22037945/21962404/8e8742fa-db25-11e6-8a9c-4e9429a00e72.png)
Can be sth simple as that. Maybe even without {avarage CP}. still it would be easy to find gym. but it won't make mess on map :D

**Comment by RezzZ** [comment]
you are absolutely right. Even a circle like this is more than enough. And than in the color of the team. Blue gyms still show the level and are perhaps bigger than pokestops so no confusion there either.

**Comment by PokeAlert** [comment]
Thansk guys. Im open to anyone icon. Size should be  100x100 png (with transparency)

**Comment by RezzZ** [comment]
@PokeAlert can you add the numbers with text yourself? for the level of the gym? Or do you need 10 per color, one for each level?

**Comment by RezzZ** [comment]
![pokestop 1](https://cloud.githubusercontent.com/assets/70499/22240081/b848582c-e219-11e6-9c33-d81bd6dc79e5.png)
![pokestop 2](https://cloud.githubusercontent.com/assets/70499/22240082/b8495556-e219-11e6-8334-af6e8a28288f.png)

something like this @PokeAlert ?
